conans/test/functional/environment/run_environment_test.py

```diff
@@ -197,3 +197,16 @@ def test_virtualrunenv(self):
 
             output = check_output_runner(command)
             self.assertIn("Hello Tool!", output)
+
+    @pytest.mark.skipif(platform.system() != "Darwin", reason="Requires DYLD_LIBRARY_PATH")
+    def test_plus_char_dyld_path(self):
```

Comment:
This should be reproducible without really needing to compile a package (slow test), isn't it? please try to do it if possible.